app/views/users/index.html.erb

```diff
@@ -1,2 +1,12 @@
-<h1>users#index</h1>
-<p>Here is a single Post for a given user you are in '/users'</p>
\ No newline at end of file
+<div class="users_view">
+  <div class="users-container">
+    <% @users.each do |user| %>
+      <div class="user-container">
+        <div class="user-details">
+          <div class="user-name"><%= user.name %></div>
+          <div class="user-posts">Posts: <%= user.posts_counter %></div>
+        </div>
+      </div>
```

Comment:
- [ ] Please add the user image to match the following wireframe as required


![image](https://user-images.githubusercontent.com/75126481/218065274-0f54a627-9733-432f-9b6b-6dce233daff3.png)